openflow15/nxt_message.go

```diff
@@ -775,14 +773,17 @@ type PacketIn2PropPacket struct {
 }
 
 func (p *PacketIn2PropPacket) Len() (n uint16) {
-	n = p.PropHeader.Len() + p.Packet.Len()
+	return uint16(p.actualLength())
+}
 
+func (p *PacketIn2PropPacket) actualLength() int {
+	n := int(p.PropHeader.Len()) + int(p.Packet.Len())
```

Comment:
If Len() just calls actualLength(), why it needs two functions?

Reply:
Function `Len` return type is `uint16`, which is defined in the interface, if the sum of the fields is exceeds 2^16, the result will overflow. Function `actualLength` return type is `int`,

Reply:
Added comments in the code.